daemon/oci_linux.go

```diff
@@ -522,6 +521,17 @@ var (
 	}
 )
 
+// inSlice tests whether a string is contained in a slice of strings or not.
+// Comparison is case insensitive
+func inSlice(slice []string, s string) bool {
+	for _, ss := range slice {
+		if strings.ToLower(s) == strings.ToLower(ss) {
```

Comment:
I think we should not make this case-insensitive for this one; volume options are case-sensitive (i.e., `:z` and `:Z` are different options). Doing this comparison case-insensitive looks to be a bug that we had

ping @cpuguy83 wdyt?